lib/split/trial.rb

```diff
@@ -63,12 +77,12 @@ def choose!(context = nil)
       elsif @experiment.has_winner?
         self.alternative = @experiment.winner
       else
-        cleanup_old_versions
+        #cleanup_old_versions
```

Comment:
Any reason to not delete entirely?

Reply:
I was still deliberating on keeping old functionality in the form of an alternate engine.